dawn/src/dawn/IIR/FieldAccessExtents.h

```diff
@@ -69,7 +68,7 @@ class FieldAccessExtents {
 
   std::optional<Extents> readAccessExtents_;
   std::optional<Extents> writeAccessExtents_;
-  Extents totalExtents_;
+  std::optional<Extents> totalExtents_;
```

Comment:
why this change?  Might it be that totalExtents is always given, while readAccess and writeAccess are really optional?

Reply:
@mroethlin Can you help me here and tell me what you intend with this change?

Reply:
The problem was that `Extents` doesn't have a default constructor, before it was constructed with zeros.

Reply:
thanks. I put it back for the moment, because this causes segfaults / the optionals are not initialized when being accessed. We can put a proper fix when adding unstructured extents.